samples/dashboards/data/mqtt-connector/demo.css

```diff
@@ -0,0 +1,138 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/dashboards/css/datagrid.css");
+@import url("https://code.highcharts.com/dashboards/css/dashboards.css");
+
+:root {
+    /* Dimensions */
+    --chart-height: 200px;
+    --log-height: 250px;
+
+    /* Colors */
+    --custom-background: var(--highcharts-neutral-color-3);
+}
+
+/* *
+ *
+ *  Layout
+ *
+ * */
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+}
+
+.cell {
+    flex: 1;
+    min-width: 20px;
+}
+
+.cell > .highcharts-dashboards-component {
+    position: relative;
+    margin: 10px;
+    padding: 10px;
+}
+
+/* *
+ *
+ *  Dashboards/Datagrid specific
+ *
+ * */
+.highcharts-dashboards-wrapper {
+    background-color: var(--highcharts-neutral-color-10);
+    padding: 10px;
+    overflow: hidden;
+}
+
+.highcharts-dashboards-component {
+    background-color: var(--custom-background);
+    max-height: var(--chart-height);
+}
+
+.highcharts-datagrid-container {
+    font-size: 12px;
+}
+
+/* *
+ *
+ *  Message log
+ *
+ * */
+#html-log {
+    background-color: var(--highcharts-neutral-color-10);
+    padding: 0;
+    margin: 0;
+    max-height: var(--log-height);
+    overflow: none;
+}
+
+#html-log .highcharts-dashboards-component-title {
+    text-align: center;
+    font-size: 18px;
+    padding: 0;
+    margin: 10px 0;
+    background-color: var(--custom-background);
+    border-bottom: 1px dotted var(--highcharts-neutral-color-20);
+}
+
+#log-content {
+    height: 90%;
+    margin: 0;
+    padding: 0 10px;
+    font: "Courier New", Courier, monospace;
```

Comment:
```suggestion
    font-family: "Courier New", Courier, monospace;
```

Reply:
Do we want to use Courier instaed of the default one?

Reply:
Personally, I like the Courier here because we simulate a console window with logs.